DependencyInjection/Configuration.php

```diff
@@ -51,6 +51,12 @@ public function getConfigTreeBuilder()
                                     ->append($this->createConsumersNodeDefinition())
                                 ->end()
                             ->end()
+                            ->arrayNode('old_rabbitmq')
```

Comment:
Ok yes, but maybe snc_redis should be renamed into redis at some time & the responsibility of this lib should be to adapt anyway.